community/modules/compute/gke-job-template/templates/gke-job-base.yaml.tftpl

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: batch/v1
```

Comment:
No action needed now, just flagging this is a candidate for parametrization.

Reply:
Ack. There are no other versions for now.